prusti-tests/tests/verify/fail/assert-false/error-position.rs

```diff
@@ -0,0 +1,49 @@
+use prusti_contracts::*;
+
+#[ensures(false)] //~ ERROR postcondition
```

Comment:
Should all of these `//~ ERROR postcondition` not include the whole `postcondition may not hold` string?

Reply:
I'll add it, why not. I think we initially just wanted to test that the program does not verify, but now we can strength the check.